docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,474 @@
+
+Refer to the following pinout diagram for your MCP3008 Analog to Digital Converter:
+
+![Pinout diagram for the ADC.](../../img/plant-watering-pi/adc-pinout.png)
+
+Consult your Raspberry Pi's data sheet or [this resource](https://pinout.xyz/pinout/3v3_power) for your board's pinout diagram.
+
+Wire the pins as follows:
+
+| MCP3008 ADC Pin | Raspberry Pi Pin |
+| ----------- | ---------------- |
+| VDD | 3.3V |
+| VREF | 3.3V |
+| AGND | GND |
+| DGND | GND |
+| CLK | SCLK |
+| DOUT | MISO |
+| DIN | MOSI |
+| CS/SHDN | GPIO25 |
+
+[**Wiring Diagram**](#wiring-diagram)
+
+### Wire your Soil Moisture Sensor
+
+Next, wire your Capacitive Soil Moisture Sensor to your Pi and ADC.
+
+Refer to the following pinout diagram for your Capacitive Soil Moisture Sensor:
+
+![Pinout diagram for the Capacitive Soil Moisture Sensor.](../../img/plant-watering-pi/moisture-sensor-pinout.png)
+
+Wire the pins as follows:
+
+| Pi | ADC |
+|--|--|
+|<table> <tr><th>Moisture Sensor Pin</th><th>Raspberry Pi Pin</th></tr><tr><td>VCC</td><td>3.3V</td></tr><tr><td>GND</td><td>GND</td></tr> </table>| <table> <tr><th>Moisture Sensor Pin</th><th>MCP3008 ADC Pin</th></tr><tr><td>A0 (Analog Signal Output)</td><td>CH0</td></tr> </table>|
+
+Put the soil moisture sensor inside of your plant box.
+
+[**Wiring Diagram**](#wiring-diagram)
+
+### Wire your Pump
+
+Now, wire and power your Peristaltic Pump [motor](/components/motor/) to complete your hardware setup.
+
+1. Connect your battery to the DC power pins on your motor speed controller.
+2. Then, connect the output pins on your motor speed controller to the pump.
+3. Connect the GND pin on the pump to the breadboard and terminal.
+4. Connect the PWM pin on the pump to [Pin 12](https://pinout.xyz/pinout/pin12_gpio18) of the Pi.
+5. Connect the plastic tubing to the pump. Put the suction end inside of your water box, and the output end inside of your plant box.
+
+### Wiring Diagram
+
+Refer to the following full wiring diagram for your hardware setup:
+
+![The full wiring diagram for all the hardware for the Plant Watering Robot.](../../img/plant-watering-pi/full-wiring.png)
+
+## Program Your Plant Watering Robot
+
+### Enable SPI on your Pi
+
+Now that you have wired your ADC and Moisture Sensor, make sure that SPI is enabled on your Pi.
+
+SSH into your Pi and run the following command:
+
+``` shell
+sudo raspi-config
+```
+
+Once the `raspi-config` interface is open, navigate to **Interface Options**:
+
+![Raspi-config Tool interface with Interface Options selected.](../../img/plant-watering-pi/interface-options.png)
+
+Then, click on **SPI**:
+
+![Raspi-config Tool interface with SPI selected.](../../img/plant-watering-pi/spi.png)
+
+Now, click **Yes** to enable SPI:
+
+![Raspi-config Tool interface with Yes selected for SPI enablement.](../../img/plant-watering-pi/spi-enabled.png)
+
+Finally, click **Finish** and restart your Pi.
+
+### Test your Soil Moisture readings on your Pi
```

Comment:
It might be nicer if we had a tutorial repository that allowed for simple install of these python dependancies as part of a modular resource installation.  Then, they could also test via the CONTROL tab vs command line.  But, we could always improve that later.

Reply:
I can bench this and the modular resource update for after I do the full setup with the parts in. That will take a bit more time for me. I was able to test all the SPI instructions & the sensor code-- I don't have the MCP3008 so it wasn't reading anything, but it does work

Reply:
LMK what you think @mcvella

Reply:
ok, sounds good